### PR TITLE
style guide consistency

### PR DESCRIPTION
**Pull request by ErinBecker**
Update things to be consistent with our style guide

### CODE REVIEW

**Comment by karenword** [comment]
Thank you @ErinBecker !

File: _episodes/20-checkout.md [hunk 1]
```diff
@@ -3,37 +3,37 @@ title: "Checkout Process"
...
 - "To certify, you must contribute to a lesson, take part in a discussion, and do a teaching demo within 90 days of your training event."
 ---
 
 During this period after lunch, we'll be talking about some of the nuts and bolts
-of getting involved in the Carpentries.  First, we'll discuss what actions you'll
-need to take after this training to become a fully certified instructor.  After that,
-we'll address any questions about the Carpentries organizations, running workshops, and
+of getting involved in The Carpentries.  First, we'll discuss what actions you'll
+need to take after this training to become a fully certified Instructor.  After that,
+we'll address any questions about The Carpentries organization, running workshops, and
```
Comment (ChristinaLK) [inline _episodes/20-checkout.md:17]:
```suggestion
we'll address any questions about The Carpentries organisation, running workshops, and
```

**Review by ChristinaLK — COMMENTED** [review]
oops, already merged.

File: _episodes/20-checkout.md (hunk 1 above)
Comment (karenword) [reply _episodes/20-checkout.md:17]: Whoops, good catch!

**Pull request merged**